frontend/src/routes/Infrastructure/Clusters/ManagedClusters/ClusterDetails/ClusterMachinePools/ClusterMachinePools.tsx

```diff
@@ -194,9 +194,7 @@ export function MachinePoolsTable() {
         if (machinePool.spec?.autoscaling) {
           actions = actions.filter((action) => action.id !== 'scaleMachinePool')
           actions = actions.filter((action) => action.id !== 'enableAutoscale')
-        }
-
-        if (machinePool.spec?.replicas) {
```

Comment:
This test did not work if the spec had `replicas: 0` which is the case for SNO. But `autoscaling` and `replicas` are mutually exclusive, so better to just check for `autoscaling`.